testutil/schema.go

```diff
@@ -130,3 +135,40 @@ func VerifySchema(t *testing.T, dg *dgo.Dgraph, opts SchemaOptions) {
 
 	CompareJSON(t, GetFullSchemaJSON(opts), string(resp.GetJson()))
 }
+
+func UpdateGQLSchema(t *testing.T, sockAddrHttp, schema string) {
```

Comment:
we already have `RetryUpdateGQLSchema`

Reply:
Not able to import anything from  graphql/e2e/common package beacuse of this error "import cycle not allowed in test"